### PR TITLE
feat: added `2020 Apache APISIX DevCon`

### LINKED ISSUE #19: add `2020 Apache APISIX DevCon` to website
source: https://www.bilibili.com/video/BV1Sv411x7gS
url: apiseven.com/resources/apisix-devcon-2020/

for example: https://www.confluent.io/resources/kafka-summit-2020/

### PR DESCRIPTION
**Pull request by guoqqqi**
resolve #19 

Todo:
- [x] contents;
- [x] responsive on Mobile; 
- [x] Header background;
- [ ] Update Nav;

look:https://api7-website-git-fork-guoqqqi-add-devcon-page-apiseven.vercel.app/zh/resources/apisix-devcon-2020

### CODE REVIEW

**Review by moonming — APPROVED** [review]

**Review by juzhiyuan — COMMENTED** [review]
There have some known issues, I'm discussing them with @guoqqqi and he will resolve them ASAP.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "editor.tabSize": 2
+}
```
Comment (juzhiyuan) [inline .vscode/settings.json:3]: Why we need this file?

File: _posts/apisix-devcon-2020/zh-CN/microblog.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (juzhiyuan) [inline _posts/apisix-devcon-2020/zh-CN/microblog.md:1]: should be `webo`

File: data/devcon.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
+{
+  "videos": {
+    "zh-CN": [
+      {
+        "title": "参会人员自我介绍",
+        "desc": "",
+        "content": "无更多介绍",
```
Comment (juzhiyuan) [inline data/devcon.json:7]: This field should be `description` or `summary` instead of `content`.

File: data/devcon.json [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        "path": "/zh/apisix-devcon-2020/self-introduction"
+      },
+      {
+        "title": "Apache APISIX 的过去和未来",
+        "desc": "温铭｜Apache APISIX PMC",
```
Comment (juzhiyuan) [inline data/devcon.json:12]: use `speaker` here

File: views/Resources/Resources.tsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+          </div>
+        </div>
+        <section>
+          <div className="container">
+            <h2>视频列表</h2>
+            <ul>
+              {list.map((item, index) => (
+                <li key={item.title}>
+                  <div className="left">
+                    <p className="title">{item.title}</p>
```
Comment (juzhiyuan) [inline views/Resources/Resources.tsx:51]: use `a` tag with `anchor linke` here.

File: views/Resources/Resources.tsx [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+                    <p className="speaker">{item.desc}</p>
+                  </div>
+                  <div className="right" id="stp"><a href={item.path} target="blank">观看视频</a></div>
```
Comment (juzhiyuan) [inline views/Resources/Resources.tsx:54]: what's `stp`?

File: views/Resources/Resources.tsx [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+                  <span id={`role${index}`}>
+                    <a className="showButton" href={`#${item.title}`}>观看描述</a>
```
Comment (juzhiyuan) [inline views/Resources/Resources.tsx:56]: what's `描述`?

**Pull request merged**